docker-compose.yml

```diff
@@ -732,27 +758,30 @@ services:
       - _APP_CONNECTIONS_QUEUE
       - _APP_REGION
 
-  appwrite-assistant:
-    container_name: appwrite-assistant
-    image: appwrite/assistant:0.1.0
```

Comment:
You can leave this uncommented. 
To make it work on mac books, you can add the `platform: linux/amd64` property